test-apps/ui-test-app/src/frontend/appui/LocalFileSupport.ts

```diff
@@ -12,22 +12,26 @@ import { IModelError } from "@bentley/imodeljs-common";
 export class LocalFileSupport {
 
   public static localFilesSupported = (): boolean => {
-    if (!SampleAppIModelApp.testAppConfiguration?.snapshotPath) {
+    if (ProcessDetector.isElectronAppFrontend)
+      return true;
+    else if (SampleAppIModelApp.testAppConfiguration?.snapshotPath)

```

Comment:
`else` is unnecessary here

Reply:
Submitted PR https://github.com/imodeljs/imodeljs/pull/941